source/element-parameters.md

```diff
@@ -66,387 +69,68 @@ Element:
 ```
 
 %---------------------------------------------------------------------------------------------------
-(s:ac.kicker.params)=
-## ACKickerP:  AC-Kicker Parameters
+## Dependent and User Set parameters.
 
-In Construction...
+Some parameters are set in the lattice file. These parameters are called "User settable". 
+Some parameters are not settable and will be computed by lattice expansion code. 
+For example, the `FloorP` parameters can be set for the `BeginningEle` element which is the
+first element of any branch line. For most other elements, the `FloorP` parameters can
+be calculated starting at the `BeginningEle` and working 
```

Comment:
This section ends abruptly.  Maybe it would be good to add "(... under construction)" or something similar.

Reply:
Added "and working forward computing the floor parameters
element-by-element."